src/components/Modals/Settings/SettingsList.tsx

```diff
@@ -87,6 +87,17 @@ export const SettingsList: FC<SettingsListProps> = ({ appHistory }) => {
     }
   }
 
+  const handleClearCacheClick = async () => {
```

Comment:
`useCallback()` if we feel like it

Reply:
good call, incoming

Reply:
https://github.com/shapeshift/web/pull/4155/commits/f44c18c9da95dad8f297f3fdea2528378e4ae22d